backend/src/timers/timer.js

```diff
@@ -0,0 +1,82 @@
+import { PrismaClient } from '@prisma/client'
+import { CalendarService } from './calendarService'
+import { TimerService } from './timerService'
+import nodemailer from 'nodemailer'
+import webpush from 'web-push'
+
+const db = new PrismaClient()
+const calendarService = new CalendarService(db)
+const timerService = new TimerService(db)
+
+const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+        user: 'nlskoolworkshop@gmail.com',
+        pass: 'Avans55!'
```

Comment:
Geen gebruikersnamen en wachtwoorden in de code neerzetten, zou dit account ook maar snel verwijderen. Er zijn bots die github scannen voor login gegevens